go.mod

```diff
@@ -84,13 +84,13 @@ require (
 	go.opentelemetry.io/proto/otlp v0.7.0
 	golang.org/x/crypto v0.0.0-20210817164053-32db794688a5
 	golang.org/x/exp v0.0.0-20210220032938-85be41e4509f // indirect
-	golang.org/x/net v0.0.0-20210825183410-e898025ed96a
+	golang.org/x/net v0.0.0-20211015210444-4f30a5c0130f
```

Comment:
follow-up, but we'll likely need to bump to master of this dep to pick up the CVE fix in https://github.com/golang/net/commit/491a49abca63de5e07ef554052d180a1b5fe2d70